models/uuv_bluerov2_heavy/uuv_bluerov2_heavy.sdf

```diff
@@ -561,8 +615,8 @@
             <hil_mode>false</hil_mode>
             <hil_state_level>false</hil_state_level>
             <vehicle_is_tailsitter>false</vehicle_is_tailsitter>
-            <send_vision_estimation>true</send_vision_estimation>
-            <send_odometry>false</send_odometry>
+            <send_vision_estimation>0</send_vision_estimation>
+            <send_odometry>1</send_odometry>
```

Comment:
Are we using the odometry somewhere?

Reply:
no, it was a test that I forgot to remove again. Changing it.